gammapy/makers/utils.py

```diff
@@ -396,12 +396,17 @@ def make_theta_squared_table(
     alpha_tot = np.zeros(len(table))
     livetime_tot = 0
 
+    if position_off is None:
```

Comment:
Could just re-write as `create_off = position_off is None`, but the existing solution is readable as well...

Reply:
right